packages/react-components/react-charts-preview/src/components/LineChart/LineChart.tsx

```diff
@@ -1280,8 +1291,9 @@ export const LineChart: React.FunctionComponent<ILineChartProps> = React.forward
       tickValues,
       tickFormat,
     };
-
+    const arrowAttributes = useArrowNavigationGroup({ axis: "horizontal" });
     return !_isChartEmpty() ? (
+      <div {...arrowAttributes}>
```

Comment:
I dont think this arrowAttributes is needed since it is already present in Cartesian chart

Reply:
Yes, ideally it should not be needed. But it is not working without that.